Pocket.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -72,6 +72,15 @@
         "revision" : "730166aa43b900e71a128e74910b1d9a431b24cf"
       }
     },
+    {
+      "identity" : "sdwebimage",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/SDWebImage/SDWebImage.git",
+      "state" : {
+        "revision" : "4178d12a44691182c2eb79c70281b14cb73f4cbf",
+        "version" : "5.14.3"
+      }
+    },
```

Comment:
how come this got added vs a version change?

Reply:
it must be a dependency of a dependency, i'm just not sure which one (i don't think spm can show a dependency tree)